src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -1671,6 +1734,10 @@ public function removeColumn($name)
             return $c['name'] != $name;
         }));
 
+        $this->commands = array_values(array_filter($this->commands, function ($c) use ($name) {
```

Comment:
You may also use an arrow function here

Reply:
Same style as the upper line, I think we should fix them both or neither.